include/spi.h

```diff
@@ -102,6 +102,7 @@ extern "C" {
 #define SPI_LINES_SINGLE	(0)
 #define SPI_LINES_DUAL		BIT(11)
 #define SPI_LINES_QUAD		BIT(12)
+#define SPI_LINES_OCTAL		BIT(13)
```

Comment:
Bit 13 is still used by `SPI_HOLD_ON_CS`, only the documentation of `spi_config` has been changed, not the bit definitions themselves.
But wouldn't it be better to stay with bits [11 : 12] assigned to SPI_LINES configuration, and define the options as below?
```
#define SPI_LINES_SINGLE	(0 << 11)
#define SPI_LINES_DUAL		(1 << 11)
#define SPI_LINES_QUAD		(2 << 11)
#define SPI_LINES_OCTAL		(3 << 11)
```
They cannot be selected together.

Reply:
True, it's like the word size. Missed that